doc/whats_new/v0.20.rst

```diff
@@ -276,6 +276,12 @@ Feature Extraction
   (words or n-grams). :issue:`9147` by :user:`Claes-Fredrik Mannby <mannby>`
   and `Roman Yurchak`_.
 
+Preprocessing
+
+- Fixed bugs in :class:`preprocessing.LabelEncoder` which would throw
```

Comment:
throw -> sometimes throw (or if you have a better way to be more specific)

Reply:
LabelEncoder always throws an error when called with empty arrays. That's why I stated 
> LabelEncoder throws error when transform or inverse_transform is called with __empty arrays.__"
 
What I believe you mean is:
> "LabelEncoder sometimes throws error on certain inputs". 

I think the former is more specific?

Reply:
> LabelEncoder always throws an error when called with empty arrays.

Why?  No error in master for 
```python
from sklearn.preprocessing import LabelEncoder
le = LabelEncoder()
le.fit([1,2])
le.transform([])
```

Reply:
opps! My bad, I'll change that